extensions/wikia/ContentReview/ContentReviewImportJS.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Wikia\ContentReview;
+
+class ImportJS {
+	const IMPORT_SCRIPTS = 'ImportJS';
+	const IMPORT_SCRIPTS_FUNCTION = 'importWikiaScriptPages';
+	const IMPORT_SCRIPTS_KEY = 'content-review-importjs';
+	const IMPORT_SCRIPTS_VERSION = '1.0';
+
+	/**
+	 * Get js script to load imports
+	 *
+	 * @return String
+	 */
+	public function getImportScripts() {
+		$importScript = \WikiaDataAccess::cache(
```

Comment:
Minor thing, but I think that if the function is called `getImportScripts`, then the variable should be called `$importScripts` to match. :)